src/frontends/tensorflow/src/translate_session.cpp

```diff
@@ -446,7 +452,8 @@ void TranslateSession::translate_graph(const ov::frontend::InputModel::Ptr& inpu
         }
 
         auto param = std::make_shared<ov::op::v0::Parameter>(input_type, input_shape);
-        set_node_name(input_name, param);
+        param->set_friendly_name(operation_name);
+        set_out_name(operation_name + ":" + std::to_string(port_index), param);
```

Comment:
Will it work if we cut by input of the node `1:node_name` by MO?

Reply:
Changed this part as discussed offline.